app/src/routes/setup/form.ts

```diff
@@ -0,0 +1,136 @@
+import { DeepPartial, Field, SetupForm } from '@directus/types';
+import { computed, ComputedRef, MaybeRef, ModelRef, Ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import z from 'zod';
+
+export const FormValidator = z.object({
+	first_name: z.string(),
+	last_name: z.string(),
+	email: z.email(),
+	password: z.string(),
+	password_confirm: z.string(),
+	project_usage: z.enum(['personal', 'commercial', 'community']).nullable().optional(),
+	org_name: z.string().nullable().optional(),
```

Comment:
I think Zod allows for custom error messages, I think we should replace the "Value can't be null" with something like "Please include organization name"

![alt text](image.png)